draft-barnes-mls-protocol.md

```diff
@@ -1435,44 +1434,128 @@ all arrive at the following state:
 
 # Message Protection
 
-[[ OPEN ISSUE: This section has initial considerations about message
-protection.  This issue clearly needs more specific recommendations,
-possibly a protocol specification in this document or a separate
-one. ]]
-
-The primary purpose of this protocol is to enable an authenticated
-group key exchange among participants.  In order to protect messages sent among
-those participants, an application will need to specify how messages are protected.
-
-For every epoch, the root key of the ratcheting tree can be used to
-derive key material for symmetric operations such as encryption/AEAD and MAC;
-AEAD or MAC MUST be used to ensure that the message originated from a member
-of the group.
-
-In addition, asymmetric signatures SHOULD be used to authenticate the sender
-of a message.
+The primary purpose of the handshake protocol is to enable an authenticated
+group key exchange among participants.  In order to protect Application messages sent
+among those participants, an application will need to specify how messages
+are protected. The message protection layer is somewhat independant of the chosen
+group key-establishment mechanism as long as this mechanism provides the
+expected cryptographic security properties for the Message Protection Layer.
```

Comment:
This section equivocates too much.  It should take a more concrete position, "This is the format for protecting application messages".  Obviously, one can use the handshake process with other application protection schemes (just as is done with [DTLS-SRTP](https://tools.ietf.org/html/rfc5764)), but this is the standard one.